components/monitoring/logging/base/configure-logging.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  name: "instance"
+  namespace: "openshift-logging"
+spec:
+  collection:
+    logs:
+      fluentd: {}
+      type: fluentd
```

Comment:
@amfred so it's ok that we are not using fluent bit?

Reply:
Yes, fluent bit wasn't working and it's not well supported yet.